moveos/moveos/src/vm/move_vm_ext.rs

```diff
@@ -174,6 +174,14 @@ where
                         Err(err) => return Err(err),
                     }
                 }
+
+                for module in &compiled_modules {
+                    let result = moveos_verifier::verifier::verify_private_generics(module);
```

Comment:
Directly use `?` to raise error? And merge the `verify_init_function` loop with `verify_private_generics`?

Reply:
I think we can do it this way. I plan to solve the problem of issue https://github.com/rooch-network/rooch/issues/244 later, and then merge `verify_init_function` and `verify_private_generics` into a function called `verify_publish_request`.